packages/components/ng-ovh-payment-method/src/components/integration/constants.js

```diff
@@ -0,0 +1,15 @@
+export const AVAILABLE_CALLBACK_STATUS_ENUM = {
+  CANCEL: 'cancel',
+  ERROR: 'error',
+  FAILURE: 'failure',
+  PENDING: 'pending',
+  SUCCESS: 'success',
+};
+
+export const DEFAULT_BINDINGS_VALUES = {
+  callbackStatusParamUrlName: 'status',
+};
+
+export default {
+  DEFAULT_BINDINGS_VALUES,
```

Comment:
Both `const` can also be exported in `default` I guess:

```suggestion
  AVAILABLE_CALLBACK_STATUS_ENUM,
  DEFAULT_BINDINGS_VALUES,
```